src/app/models/node-utilization.model.ts

```diff
@@ -25,4 +28,42 @@ export class NodeUtilization {
       nodeNames: null | string[];
     }[],
   ) {}
+
+  // transform NodeUtilization to NodeUtilizationChartData for NodeUtilization bar chart
+  toNodeUtilizationChartData(){
+    const MAX_NODES_IN_DESCRIPTION = 15;
+    const backgroundColor = CHART_COLORS[0];
+    let type = this.type;
+    let utilization = this.utilization;
+    // prepare data
+    let chartDataItems = new Array<ChartDataItem>();
+    let index = 0;
```

Comment:
This variable is declared but never used.

Reply:
Just removed, sorry for missing that.